Cython/Compiler/Nodes.py

```diff
@@ -4762,6 +4762,7 @@ class PyClassDefNode(ClassDefNode):
     metaclass = None
     mkw = None
     doc_node = None
+    orig_bases = None
```

Comment:
This also needs to be added to `child_attrs` to make sure it gets properly traversed. `CloneNode` blocks the traversal.
And please also document it in the comments above.